### ISSUE TITLE
Singular procedure bindings broken

### ISSUE THREAD

**Issue by b-reinke** [sagemath/sage#37414]
### Steps To Reproduce

I want to use the singular binding for multivariate polynomials in order to compute radicals or elimination ideals.

Already the example of the documentation doesn't work:
```

      sage: R.<x,y,z> = PolynomialRing(QQ, 3)
      sage: I = (x^2, y^3, (x*z)^4 + y^3 + 10*x^2) * R
      sage: I.radical()
```

### Expected Behavior

```
      Ideal (y, x) of Multivariate Polynomial Ring in x, y, z over Rational Field
```

### Actual Behavior

```
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
File /usr/lib/python3.11/site-packages/sage/libs/singular/function.pyx:1777, in sage.libs.singular.function.singular_function (build/cythonized/sage/libs/singular/function.cpp:24582)()
   1776 try:
-> 1777     return SingularKernelFunction(name)
   1778 except NameError:

File /usr/lib/python3.11/site-packages/sage/libs/singular/function.pyx:1578, in sage.libs.singular.function.SingularKernelFunction.__init__ (build/cythonized/sage/libs/singular/function.cpp:24170)()
   1577 super().__init__(name)
-> 1578 self.call_handler = self.get_call_handler()
   1579 

File /usr/lib/python3.11/site-packages/sage/libs/singular/function.pyx:1584, in sage.libs.singular.function.SingularKernelFunction.get_call_handler (build/cythonized/sage/libs/singular/function.cpp:24299)()
   1583 if not cmd_n:
-> 1584     raise NameError("Singular kernel function {!r} is not defined".format(self._name))
   1585 

NameError: Singular kernel function 'radical' is not defined

During handling of the above exception, another exception occurred:

ValueError                                Traceback (most recent call last)
Cell In[11], line 3
      1 R = PolynomialRing(QQ, Integer(3), names=('x', 'y', 'z',)); (x, y, z,) = R._first_ngens(3)
      2 I = (x**Integer(2), y**Integer(3), (x*z)**Integer(4) + y**Integer(3) + Integer(10)*x**Integer(2)) * R
----> 3 I.radical()

File /usr/lib/python3.11/site-packages/sage/rings/polynomial/multi_polynomial_ideal.py:304, in RequireField.__call__(self, *args, **kwds)
    302 if not R.base_ring().is_field():
    303     raise ValueError("Coefficient ring must be a field for function '%s'." % (self.f.__name__))
--> 304 return self.f(self._instance, *args, **kwds)

File /usr/lib/python3.11/site-packages/sage/libs/singular/standard_options.py:143, in libsingular_gb_standard_options.<locals>.wrapper(*args, **kwds)
    139 """
    140 Execute function in ``LibSingularGBDefaultContext``.
    141 """
    142 with LibSingularGBDefaultContext():
--> 143     return func(*args, **kwds)

File /usr/lib/python3.11/site-packages/sage/rings/polynomial/multi_polynomial_ideal.py:1766, in MPolynomialIdeal_singular_repr.radical(self)
   1724 r"""
   1725 The radical of this ideal.
   1726 
   (...)
   1763      of Multivariate Polynomial Ring in x, y, z over Finite Field of size 37
   1764 """
   1765 from sage.libs.singular.function_factory import ff
-> 1766 radical = ff.primdec__lib.radical
   1767 r = radical(self)
   1769 S = self.ring()

File /usr/lib/python3.11/site-packages/sage/libs/singular/function_factory.py:40, in SingularFunctionFactory.__getattr__(self, name)
     37     raise AttributeError("Singular Function Factory has no attribute '%s'" % name)
     39 try:
---> 40     return singular_function(name)
     41 except NameError:
     42     if name.endswith("__lib"):

File /usr/lib/python3.11/site-packages/sage/libs/singular/function.pyx:1779, in sage.libs.singular.function.singular_function (build/cythonized/sage/libs/singular/function.cpp:24622)()
   1777         return SingularKernelFunction(name)
   1778     except NameError:
-> 1779         return SingularLibraryFunction(name)
   1780 
   1781 def lib(name):

File /usr/lib/python3.11/site-packages/sage/libs/singular/function.pyx:1531, in sage.libs.singular.function.SingularLibraryFunction.__init__ (build/cythonized/sage/libs/singular/function.cpp:23728)()
   1529     """
   1530     super().__init__(name)
-> 1531     self.call_handler = self.get_call_handler()
   1532 
   1533 cdef BaseCallHandler get_call_handler(self) noexcept:

File /usr/lib/python3.11/site-packages/sage/libs/singular/function.pyx:1538, in sage.libs.singular.function.SingularLibraryFunction.get_call_handler (build/cythonized/sage/libs/singular/function.cpp:23880)()
   1536     raise NameError("Singular library function {!r} is not defined".format(self._name))
   1537 if singular_idhdl.typ!=PROC_CMD:
-> 1538     raise ValueError("Not a procedure")
   1539 
   1540 cdef LibraryCallHandler res = LibraryCallHandler()

ValueError: Not a procedure
```

### Environment

```markdown
- **OS**: Arch Linux
- **Sage Version**: 10.2-2
- **Singular Version**: 4.3.2.p15-1
```

### Checklist

- [X] I have searched the existing issues for a bug report that matches the one I want to file, without success.
- [X] I have read the documentation and troubleshoot guide

**Comment by yyyyx4** [comment]
@antonio-rojas, this looks like it may be a problem with the Arch package..?

**Comment by antonio-rojas** [comment]
Yes, this got broken by the latest singular update. Fixed in 10.2-3 (but this will eventually bite upstream Sage too)
